spec/swagger_schemas/filters/list_applications.rb

```diff
@@ -0,0 +1,55 @@
+LIST_APPLICATIONS_FILTER = {
+  v1: {
```

Comment:
As these schema versions are all very similar between versions could we construct the hash in a way that DRYs them up and makes it clear what the differences are between versions? Something like:

```
LIST_APPLICATIONS_FILTER = {
  v1: {
    ...
  }
}.tap do |h|
  h[v2] =  h[v1].merge({ 
    properties: { cohort: { description: "Overrides description" } 
  })
end.freeze
```